quiche/src/lib.rs

```diff
@@ -3890,7 +3890,12 @@ impl Connection {
         // Create PING for PTO probe if no other ack-eliciting frame is sent or if
         // we've sent too many non ACK eliciting packets without having
         // sent an ACK eliciting one
```

Comment:
Might worth updating this comment to cover the new condition, possibly we can break the long paragraph into some bullet points or something for clarity